test/raw.js

```diff
@@ -339,6 +349,16 @@ describe('bodyParser.raw()', function () {
       test.expect(200, 'buf:6e616d653de8aeba', done)
     })
 
+    if (hasBrotliSupport) {
```

Comment:
Instead of excluding the test, have the test always be there but show up as skipped in the mocha output when the support is missing.